digiwf-cypress/cypress/components/nav.js

```diff
@@ -0,0 +1,118 @@
+    currentInstancesBtn: () =>
+      cy.get('nav [data-test="current-instances-btn"]'),
+    startProcessBtn: () => cy.get('nav [data-test="start-process-btn"]'),
+    openGroupTasksBtn: () => cy.get('nav [data-test="open-group-tasks-btn"]'),
+    assignedGroupTasksBtn: () =>
+      cy.get('nav [data-test="assigned-group-tasks-btn"]'),
+  };
+
+  openMyTasks() {
+    this.elements
+      .myTasksBtn()
+      .click()
+      .then(() => {
+        myTasks.checkHeadline();
+        startProcess._waitUpdate("@dataGetMyTasks");
+      });
+    return myTasks;
+  }
+
+  openCurrentInstances() {
+    this.elements
+      .currentInstancesBtn()
+      .click()
+      .then(() => {
+        currentInstances.checkHeadline();
+        startProcess._waitUpdate("@dataGetInstances");
+      });
+    return currentInstances;
+  }
+
+  openStartProcess() {
+    this.elements
+      .startProcessBtn()
+      .click()
+      .then(() => {
+        startProcess.checkHeadline();
+        startProcess._waitUpdate("@dataGetDefinitions");
+      });
+    return startProcess;
+  }
+
+  openOpenGroupTasks() {
+    this.elements
+      .openGroupTasksBtn()
+      .click()
+      .then(() => {
+        openGroupTasks.checkHeadline();
+        startProcess._waitUpdate("@dataGetOpenGroupTasks");
+      });
+    return openGroupTasks;
+  }
+
+  openInProgressGroupTasks() {
+    this.elements
+      .assignedGroupTasksBtn()
+      .click()
+      .then(() => {
+        inProgressGroupTasks.checkHeadline();
+        startProcess._waitUpdate("@dataGetAssignedGroupTasks");
+      });
+    return inProgressGroupTasks;
+  }
+
+  /**
+   * Collects task counts for different pages and saves them via cypress alias.
+   * @param prefix The prefix to use for the alias name.
+   * @param types A list of pages to collect the metrics for. Allowed values are "myTasks", "openGroupTasks" and "inProgressGroupTasks". Default is all of them.
+   */
+  gatherTaskMetrics(
+    prefix,
+    types = ["myTasks", "openGroupTasks", "inProgressGroupTasks"]
+  ) {
+    if (types.includes("myTasks")) {
+      let myTasks = this.openMyTasks();
+      myTasks.getItemCount().as(prefix + "_myTasksCount");
+    }
+    if (types.includes("openGroupTasks")) {
+      let openGroupTasks = this.openOpenGroupTasks();
+      openGroupTasks.getItemCount().as(prefix + "_openGroupTasksCount");
+    }
+    if (types.includes("inProgressGroupTasks")) {
+      let inProgressGroupTasks = this.openInProgressGroupTasks();
+      inProgressGroupTasks
+        .getItemCount()
+        .as(prefix + "_inProgressGroupTasksCount");
+    }
+  }
+
+  /**
+   * Compares two via {@link gatherTaskMetrics} collected task counts.
+   * @param prefix1 Prefix used for the first collection.
+   * @param prefix2 Prefix used for the second collection.
+   * @param differences Map of differences between first and second collections.
+   */
+  compareTaskMetrics(prefix1, prefix2, differences) {
```

Comment:
Da ich es erst nicht verstanden hatte und anderen immer gerne Zeit sparen möchte, vielleicht noch ergänzen:

```suggestion
  /**
   * Compares two via {@link gatherTaskMetrics} collected task counts with regard to their difference.
   * This is necessary because a number might already be present before the test.
   * @param prefix1 Prefix used for the first collection.
   * @param prefix2 Prefix used for the second collection.
   * @param differences Map of differences between first and second collections.
   */
  compareTaskMetrics(prefix1, prefix2, differences) {
```

Reply:
Das ist eigentlich nicht wirklich der Grund sondern um zu testen, dass die Tasks auf den entsprechenden Seiten angezeigt werden und es einer mehr wurde, ob davor welche vorhanden sind oder nicht ist theoretisch egal.

Reply:
Ich glaube, dass wir das gleiche meinen.
Zuerst hatte ich mich eben gewundert, warum man nicht einfach sowas macht wie `taskMetricsIs`. Das geht aber nicht, weil ja schon vorher welche da sein könnten und man deswegen die Differenz prüfen muss. Meinte nur, sowas in der Art könnte man noch kurz und knapp erklären, damit nicht andere später wieder darüber stolpern.

Reply:
Habs im MR #1783 mit 1657e5eecb56aaa83c073ecfdf4700594bb58e6e angepasst.